README.rst

```diff
@@ -100,12 +100,30 @@ This will add these paths to Django:
 
 You can use them like this in your django templates:
```

Comment:
```suggestion
```suggestion
Below is sample Django template code to use these paths depending if
you'd like to use GET or POST requests. Logging out was deprecated in
`Django 4.1 <https://docs.djangoproject.com/en/5.1/releases/4.1/#features-deprecated-in-4-1>`_.
```
```